# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1332950907 25200
#      Wed Mar 28 09:08:27 2012 -0700
# Node ID 70387b464c80cead0683ebe6e3af732b52a56dc6
# Parent  ff4143d9fc0533233a10279fd31d5c401c102fbd
doc: Fix typo in logspace docstring (bug #36024)

* logspace.m: Fix typo in logspace docstring (bug #36024)

diff --git a/scripts/general/logspace.m b/scripts/general/logspace.m
--- a/scripts/general/logspace.m
+++ b/scripts/general/logspace.m
@@ -13,17 +13,17 @@
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {} logspace (@var{a}, @var{b})
-## @deftypefnx {Function File} {} logspace (@var{b}, @var{b}, @var{n})
+## @deftypefnx {Function File} {} logspace (@var{a}, @var{b}, @var{n})
 ## @deftypefnx {Function File} {} logspace (@var{a}, pi, @var{n})
 ## Return a row vector with @var{n} elements logarithmically spaced from
 ## @tex
 ## $10^{a}$ to $10^{b}$.
 ## @end tex
 ## @ifnottex
 ## 10^@var{a} to 10^@var{b}.
 ## @end ifnottex
